android/java/res/xml/brave_main_preferences.xml

```diff
@@ -28,12 +28,6 @@
         android:icon="@drawable/ic_crypto_wallets"
         android:order="3"
         android:title="@string/brave_ui_brave_wallet"/>
-     <org.chromium.components.browser_ui.settings.ChromeBasePreference
-        android:fragment="org.chromium.chrome.browser.settings.BraveNewsPreferences"
-        android:key="brave_news"
```

Comment:
I think maybe this is supposed to be `brave_news_v2` ? (the one below this)

Reply:
Nah, we're removing the flag, so it's as if the flag is always on (i.e. the V1 stuff is dead code).